supplements/extra-life/adventure-with-muk.xml

```diff
@@ -0,0 +1,182 @@
+			<set name="wisdom">12</set>
+			<set name="charisma">6</set>
+			<set name="ac">13</set>
+			<set name="hp">10</set>
+			<set name="speed">20 ft., climb 20 ft., fly 30 ft.</set>
+			<set name="languages">—</set>
+			<set name="type">Beast</set>
+			<set name="size">Tiny</set>
+			<set name="alignment">unaligned</set>
+			<set name="challenge">1/4</set>
+			<set name="traits">ID_WOTC_AWM_COMPANION_TRAIT_BIRDSQUIRREL_PACK_TACTICS</set>
+			<set name="actions">ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_MULTIATTACK,ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_BITE,ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_CLAWS</set>
+		</setters>
+		<rules>
+			<stat name="companion:ac" value="13" />
+			<stat name="companion:hp:max" value="10" bonus="base" />
+			<stat name="companion:speed" value="20" bonus="base" />
+			<stat name="companion:speed:climb" value="20" bonus="base" />
+			<stat name="companion:speed:fly" value="30" bonus="base" />
+		</rules>
+	</element>
+	<element name="Pack Tactics" type="Companion Trait" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_TRAIT_BIRDSQUIRREL_PACK_TACTICS">
+		<description>
+			<p>Birdsquirrel has advantage on an attack roll against a creature if at least one of the Birdsquirrel's allies is within 5 feet of the creature and the ally isn't incapacitated.</p>
+		</description>
+		<sheet>
+			<description>Birdsquirrel has advantage on an attack roll against a creature if at least one of the Birdsquirrel's allies is within 5 feet of the creature and the ally isn't incapacitated.</description>
+		</sheet>
+	</element>
+	<element name="Multiattack" type="Companion Action" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_MULTIATTACK">
+		<description>
+			<p>Birdsquirrel makes two attacks: one with its bite and one with its claws.</p>
+		</description>
+		<sheet>
+			<description>Birdsquirrel makes two attacks: one with its bite and one with its claws.</description>
+		</sheet>
+	</element>
+	<element name="Bite" type="Companion Action" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_BITE">
+		<description>
+			<p>Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 3 (1d6 + 2) piercing damage.</p>
+		</description>
+		<sheet>
+			<description>Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 3 (1d6 + 2) piercing damage.</description>
+		</sheet>
+	</element>
+	<element name="Claws" type="Companion Action" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_ACTION_BIRDSQUIRREL_CLAWS">
+		<description>
+			<p>Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 1 (1d4 + 2) slashing damage.</p>
+		</description>
+		<sheet>
+			<description>Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 1 (1d4 + 2) slashing damage.</description>
+		</sheet>
+	</element>
+	<element name="Boontu Monkey" type="Companion" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_BOONTU_MONKEY">
+		<supports>Familiar, Variant Familiar</supports>
```

Comment:
also i know i did basically said you should do it, but after discussing it with others, we think we shouldn't place these supports on new creatures, unless they specifically mention being Familiars. What we gonna instead try to do in a separate pull is to change how "Familiar Selection" item works, so it would automatically grab any creature that's size Tiny.
```suggestion
```